src/com/Jacksonnn/DCCore/DCPlayerManager.java

```diff
@@ -0,0 +1,149 @@
+package com.Jacksonnn.DCCore;
+
+import com.Jacksonnn.DCCore.Storage.DCPlayerQueries;
+import com.Jacksonnn.DCCore.Storage.Mysql;
+import com.Jacksonnn.DCCore.Storage.SqlLite;
+import org.bukkit.Bukkit;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.UUID;
+
+public class DCPlayerManager {
+    private DCCore plugin;
+
+    public DCPlayerManager(DCCore plugin) {
+        this.plugin = plugin;
+    }
+
+    public void createDCPlayersTable() throws SQLException {
+        if (plugin.getDatabaseManager().getDatabase() instanceof Mysql) {
+            plugin.getDatabaseManager().getDatabase().getConnection().createStatement().execute(DCPlayerQueries.CREATE_TABLE_DCPLAYERS.getMysqlQuery());
+        } else {
+            plugin.getDatabaseManager().getDatabase().getConnection().createStatement().execute(DCPlayerQueries.CREATE_TABLE_DCPLAYERS.getSqliteQuery());
+        }
+    }
+
+    public void createDCPlayer(DCPlayer dcPlayer) {
+        GeneralMethods.addDCPlayer(dcPlayer);
+
+        String query;
+        if (plugin.getDatabaseManager().getDatabase() instanceof SqlLite) {
+            query = DCPlayerQueries.CREATE_DCPLAYER.getSqliteQuery();
+        } else {
+            query = DCPlayerQueries.CREATE_DCPLAYER.getMysqlQuery();
+        }
+        try {
+            PreparedStatement preparedStatement = plugin.getDatabaseManager().getDatabase().getConnection().prepareStatement(query);
+            preparedStatement.setString(1, dcPlayer.getUuid().toString());
+            preparedStatement.setString(2, dcPlayer.getName());
+            preparedStatement.setLong(3, dcPlayer.getPlayTime());
+            preparedStatement.setString(4, GeneralMethods.booleanToString(dcPlayer.isQuickdeposit()));
+            preparedStatement.setString(5, dcPlayer.getChatMode().getChatName());
```

Comment:
intentionally starting at 1, which doesn’t completely make sense to me either. it’s how it’s done otherwise and the data works and is the correct columns without errors. tbh i’ve never really noticed it never starts at 0